ubuntu-dev/ubuntu-dev.dockerfile

```diff
@@ -178,16 +178,13 @@ ENV PATH="${PATH}:/home/user/.cargo/bin"
 RUN rustup install nightly \
  && rustup completions bash | sudo tee /etc/bash_completion.d/rustup.bash-completion > /dev/null
 
-# Install the latest Rust Language Server.
-RUN rustup component add rls-preview \
- && rustup component add rust-analysis \
- && rustup component add rust-src \
+# Install additional Rust components.
```

Comment:
Not: I'd have preferred a more explicit comment like "Install the latest Rust Language Server and rustfmt.".

Reply:
The meaning of these components can vary (src is useful outside RLS), and we sometimes forget to update the description. I think a generic version works better here.